include/experimental/__p1673_bits/layout_blas_general.hpp

```diff
@@ -47,6 +47,10 @@
 #include "maybe_static_size.hpp"
 #include "layout_tags.hpp"
 
+#include <experimental/__p0009_bits/macros.hpp>
```

Comment:
The P1673 implementation shouldn't be depending on P0009 implementation details.  Would you consider instead just including the main `mdspan` header, <experimental/mdspan>?  Thanks!